Easydict/objc/ViewController/View/WordResultView/EZWordResultView.m

```diff
@@ -74,809 +74,808 @@ - (void)refreshWithResult:(EZQueryResult *)result {
-            
-            if (!tagScrollView) {
-                tagScrollView = [[NSScrollView alloc] init];
-                [self addSubview:tagScrollView];
-                [tagScrollView mas_makeConstraints:^(MASConstraintMaker *make) {
-                    make.left.offset(leftMargin);
-                    make.height.mas_equalTo(newSize.height);
-                    CGFloat topOffset = kVerticalMargin_12 + 3;
-                    if (lastView) {
-                        make.top.equalTo(lastView.mas_bottom).offset(topOffset);
-                    } else {
-                        make.top.offset(topOffset);
+
+            NSString *text = nil;
+            if (errorMessage.length) {
+                text = errorMessage;
+            } else if (result.translatedText) {
+                // Note that translatedText may be returned @"" by service, like Youdao, or kimi stream when censored.
+                text = result.translatedText;
+            } else {
+                text = NSLocalizedString(@"no_results_found", nil);
+            }
+
+            if (text) {
+                EZLabel *resultLabel = [[EZLabel alloc] init];
+                resultLabel.font = [NSFont systemFontOfSize:14 * self.fontSizeRatio];
+                [self addSubview:resultLabel];
+
+                // OpenAI result text has its own paragraph style.
+                if ([result.serviceTypeWithUniqueIdentifier isEqualToString:EZServiceTypeOpenAI]) {
+                    resultLabel.paragraphSpacing = 0;
+                }
+
+                resultLabel.text = text;
+                resultLabel.delegate = self;
+
+                [resultLabel mas_remakeConstraints:^(MASConstraintMaker *make) {
+                    CGFloat rightOffset = kHorizontalMargin_8;
+                    make.right.equalTo(self).offset(-rightOffset);
+                    exceptedWidth += rightOffset;
+
+                    CGFloat topOffset = explainTextFieldTopOffset + result.translateResultsTopInset;
+
+                    if (!explainLabel) {
+                        if (lastView) {
+                            make.top.equalTo(lastView.mas_bottom).offset(topOffset);
+                        } else {
+                            make.top.equalTo(self).offset(topOffset);
+                        }
+
+                        CGFloat leftPadding = kHorizontalMargin_8;
+                        exceptedWidth += leftPadding;
+                        make.left.equalTo(self).offset(leftPadding);
                     }
-                    height += topOffset;
-                    height += newSize.height;
+
+                    CGSize labelSize = [self labelSize:resultLabel exceptedWidth:exceptedWidth];
+                    make.size.mas_equalTo(labelSize).priorityHigh();
+
+                    // ???: This means the label text has more than 2 lines, so we need to adjust the top offset.
+                    if (labelSize.height > explainLabel.height * 2) {
```

Comment:
Maybe consider comment out or remove this logic.

Reply:
Removed.